### PR TITLE
Update creating-secrets.adoc

### PR DESCRIPTION
**Pull request by ernesgonzalez33**
Remove the "personal" instructions from the access tokens in GitLab to a "project" access token

### CODE REVIEW

**Review by arewm — APPROVED** [review]

**Comment by arewm** [comment]
@ernesgonzalez33 , is it possible to have a project access token that has access to more than one repository? Are there any caveats that we need to add?

**Comment by ernesgonzalez33** [comment]
> @ernesgonzalez33 , is it possible to have a project access token that has access to more than one repository? Are there any caveats that we need to add?

@arewm I added more info. WDYT?

File: docs/modules/ROOT/pages/how-tos/configuring/creating-secrets.adoc [hunk 1]
```diff
@@ -93,6 +94,8 @@ Using the PAT authentication requires only the `password` key. The `username` sh
 
 This secret will be used by the build service to perform builds with Pipeline-as-Code.
 
+This Project Access Tokens are scoped to a project, so you cannot use them to access resources from other projects. For more information regarding Project Access Tokens, see the Gitlab https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html[documentation].
```
Comment (arewm) [inline docs/modules/ROOT/pages/how-tos/configuring/creating-secrets.adoc:97]: What implications does this have if users want to onboard multiple projects? How can this be accomplished?

Comment (ernesgonzalez33) [reply docs/modules/ROOT/pages/how-tos/configuring/creating-secrets.adoc:97]: That is already in the docs written just below these lines

Comment (arewm) [reply docs/modules/ROOT/pages/how-tos/configuring/creating-secrets.adoc:97]: Ah, thanks.

**Review by arewm — APPROVED** [review]

**Pull request merged**